gtests/test_nopython_transform.cpp

```diff
@@ -116,51 +135,57 @@ class InverseTest : public ::testing::Test
     }
 };
 
-typedef ::testing::Types<float, double> TestTypes;
+typedef ::testing::Types<
+    FFTTestParams<float, true>,
+    FFTTestParams<float, false>,
+    FFTTestParams<double, true>,
+    FFTTestParams<double, false>>
+    TestTypes;
```

Comment:
Declare different test cases by different floating point types and different input data length.